extensions/spring-di/deployment/src/main/java/io/quarkus/spring/di/deployment/SpringDIProcessor.java

```diff
@@ -418,6 +427,13 @@ Set<AnnotationInstance> getAnnotationsToAdd(
                         CDI_INJECT_ANNOTATION,
                         target,
                         Collections.emptyList()));
+                // in Spring List<SomeBean> means that all instances of SomeBean should be injected
+                if (methodInfo.returnType().name().equals(DotNames.LIST)) {
```

Comment:
`methodInfo.returnType()`? I'm a bit confused now ;-)

Reply:
Oops, wrong change

Reply:
Actually is it correct. It just look weird in the UI